### PR TITLE
bug: Disable sale artworks clear-all-filters button if 0 filters selected

### PR DESCRIPTION
**Pull request by bhoggard**
The type of this PR is: BUG

This PR resolves [CX-720]

### Description

Screen Shot 2020-10-26 at 3 51 10 PM

Screen Shot 2020-10-26 at 3 51 30 PM

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[CX-720]: https://artsyproduct.atlassian.net/browse/CX-720

### CODE REVIEW

**Comment by MounirDhahri** [comment]
Sorry for the late review 🙏 I realized I did not submit my comment from yesterday

File: src/lib/Components/FilterModal/FilterModal.tsx [hunk 1]
```diff
@@ -214,6 +214,8 @@ export const FilterOptions: React.FC<FilterOptionsProps> = (props) => {
 
   const concreteAggregations = state.aggregations ?? []
 
+  const isClearAllButtonEnabled = state.appliedFilters.length > 0 || state.appliedFilters.length > 0
```
Comment (pepopowitz) [inline src/lib/Components/FilterModal/FilterModal.tsx:217]:
```suggestion
  const isClearAllButtonEnabled = state.appliedFilters.length > 0 || state.selectedFilters.length > 0
```

Based on the conversation, I _think_ this is what you had in mind here. Right now it's checking the same value twice.

**Review by pepopowitz — CHANGES_REQUESTED** [review]
I think there's a typo!

File: src/lib/Components/FilterModal/FilterModal.tsx (hunk 1 above)
Comment (bhoggard) [reply src/lib/Components/FilterModal/FilterModal.tsx:217]: oops thanks!

**Review by pepopowitz — APPROVED** [review]
❤️ ❤️ ❤️

**Pull request merged**